packages/frontend/src/features/sqlRunner/components/LineChartAggregationConfig.tsx

```diff
@@ -0,0 +1,105 @@
+import { MetricType, type YLayoutOptions } from '@lightdash/common';
+import { Box, Group, Select, Text } from '@mantine/core';
+import {
+    IconMathFunction,
+    IconMathMax,
+    IconMathMin,
+    IconMathOff,
+    IconNumber1,
+    IconSum,
+    IconTrendingUp,
+} from '@tabler/icons-react';
+import { capitalize } from 'lodash';
+import { forwardRef, type ComponentPropsWithoutRef, type FC } from 'react';
+import MantineIcon from '../../../components/common/MantineIcon';
+
+const AggregationIcon: FC<{ aggregation: string | undefined }> = ({
+    aggregation,
+}) => {
+    let icon;
+    switch (aggregation) {
+        case MetricType.COUNT:
+            icon = IconMathFunction;
+            break;
+        case MetricType.SUM:
+            icon = IconSum;
+            break;
+        case MetricType.AVERAGE:
+            icon = IconTrendingUp;
+            break;
+        case MetricType.MIN:
+            icon = IconMathMin;
+            break;
+        case MetricType.MAX:
+            icon = IconMathMax;
+            break;
+        case 'first':
+            icon = IconNumber1;
+            break;
+        default:
+            icon = IconMathOff;
+    }
+
+    return <MantineIcon color="indigo.4" icon={icon} />;
+};
+
+type Props = {
+    options: YLayoutOptions['aggregationOptions'] | undefined;
+    aggregation: YLayoutOptions['aggregationOptions'][number] | undefined;
+    onChangeAggregation: (
+        value: YLayoutOptions['aggregationOptions'][number],
+    ) => void;
+};
+
+const AggregationItem = forwardRef<
+    HTMLDivElement,
+    ComponentPropsWithoutRef<'div'> & { value: string; selected: boolean }
+>(({ value, ...others }, ref) => (
+    <Box ref={ref} {...others}>
+        <Group noWrap spacing="xs">
+            <AggregationIcon aggregation={value} />
+            <Text>{capitalize(value)}</Text>
+        </Group>
+    </Box>
+));
+
+export const LineChartAggregationConfig: FC<Props> = ({
```

Comment:
cant we reuse the current `AggregationConfig` component? I think it's the same, or not?

Reply:
Yup. I just copied these wholesale, but now Im deduping them.